apps/sms/js/activity_handler.js

```diff
@@ -294,70 +293,56 @@ var ActivityHandler = {
       };
 
     if (Compose.isEmpty()) {
-      this.launchComposer(activity);
-    } else {
-      // ask user how should we do
-      ActivityHandler.displayUnsentConfirmation(activity);
+      return this.launchComposer(activity);
```

Comment:
I usually don't like having a return in the middle of a function, but the function is small here so why not.